languages/csharp/exercises/concept/nullability/.meta/Example.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+public static class Badge
+{
+    public static int ComputeWidthPx(string? text,
+                                     int? fontSizePx)
+    {
+
+        return (text??"").Length*(fontSizePx??0);
+    }
+
+    public static string ComputeNameText(string? firstName,
+                                         string? middleName,
+                                         string? familyName)
```

Comment:
why? to me it's just like my `name` in github, I can remove it whenever I want.

Reply:
This was my inspiration to this exercise: https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Some people do not have first names or family names ;)

Reply:
Right. I understand. The thing I would like this exercise to show is using both nullable and non-nullable variants of both reference and value types. Maybe we need to tweak the exercise a bit. What if instead of the three name components, a person has just a name (one name that is) and optionally a title (dr./MSc, etc.)?

Reply:
Closing this conversation, as I assume we'll agree on a new story before changing the code.